queries.sql

```diff
@@ -59,4 +59,32 @@ GROUP BY species;
 
 SELECT species, AVG(ESCAPE_ATTEMPTS) AS avg_escapes FROM animals
 WHERE DATE_OF_BIRTH BETWEEN '1990-01-01' AND '2000-12-31'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+SELECT name FROM animals a 
+JOIN owners o ON a.owners_id = o.id 
+WHERE o.full_name = 'Melody Pond';
+
+SELECT * FROM animals a 
+JOIN species s ON a.species_id = s.id 
+WHERE s.name = 'Pokemon';
+
+SELECT full_name as owner, name as animal FROM owners
+LEFT JOIN animals
+ON owners.ID = animals.OWNERS_ID 
+
+SELECT COUNT(*), s.name FROM animals
+JOIN species s ON animals.species_id = s.id
+GROUP BY s.name;
+
+SELECT a.name FROM animals a
+JOIN species s ON a.species_id = s.id
+JOIN owners o ON a.owners_id = o.id
+WHERE s.name ='Digimon' AND o.full_name = 'Jennifer Orwell'
+
+SELECT a.name FROM animals a
+JOIN owners o ON a.owners_id = o.id
+WHERE a.escape_attempts = 0 AND o.full_name = 'Dean Winchester'
+
+
```

Comment:
- [ ] Nice job demonstrating the screenshot for the query: _**Who owns the most animals?**_ Most importantly however, let's ensure that this query also present in the `queries.sql`

Reply:
I forgot the last one. Thanks!

Reply:
I forgot the last one. Thanks!